web3-react/src/libs/injected.ts

```diff
@@ -0,0 +1,23 @@
+import { initializeConnector } from '@web3-react/core'
+import { MetaMask } from '@web3-react/metamask'
+
+import { Connection, ConnectionType, onConnectionError } from './connections'
+
+export function buildInjectedConnector() {
+  let metaMaskErrorHandler: (error: Error) => void | undefined
```

Comment:
What is the purpose of `metaMaskErrorHandler`? Doesn't look like it's being used anywhere.